spec/numberformat.html

```diff
@@ -457,16 +540,33 @@
       <emu-alg>
         1. Let _nf_ be *this* value.
         1. If Type(_nf_) is not Object, throw a *TypeError* exception.
-        1. If _nf_ does not have an [[initializedNumberFormat]] internal slot, throw a *TypeError* exception.
-        1. If _nf_.[[boundFormat]] is *undefined*, then
+        1. If _nf_ does not have an [[initializedNumberFormat]] internal slot whose value is *true*, throw a *TypeError* exception.
+        1. If the [[boundFormat]] internal slot of _nf_ is *undefined*, then
           1. Let _F_ be a new built-in function object as defined in Number Format Functions (<emu-xref href="#sec-number-format-functions"></emu-xref>).
           1. Let _bf_ be BoundFunctionCreate(_F_, _nf_, « »).
           1. Perform ! DefinePropertyOrThrow(_bf_, `"length"`, PropertyDescriptor {[[Value]]: 1, [[Writable]]: *false*, [[Enumerable]]: *false*, [[Configurable]]: *true*}).
-          1. Set _nf_.[[boundFormat]] to bf.
+          1. Set _nf_.[[boundFormat]] to _bf_.
         1. Return _nf_.[[boundFormat]].
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-intl.numberformat.prototype.formattoparts">
+      <h1>Intl.NumberFormat.prototype.formatToParts ([ value ])</h1>
```

Comment:
It would be nice for `value` to be truly non-optional, but the ship's sailed on having JS APIs that enforce arity.  So I think _only_ that the length should be 1, not 0.  `Intl.NumberFormat().formatToParts()` would act as if `NaN` were passed, even though it's dumb and doesn't make much sense, because it's how JS APIs generally work, that failure to provide an argument is identical to passing `undefined` for that argument.